src/libPMacc/include/memory/buffers/MultiGridBuffer.hpp

```diff
@@ -220,7 +221,7 @@ class MultiGridBuffer
 
     GridBuffer<Type, DIM>& getGridBuffer(typename BufferNames::Names name)
     {
-        assert(name >= 0 && name < BufferNames::Count);
+        PMACC_ASSERT(name >= 0 && name < BufferNames::Count);
```

Comment:
@psychocoderHPC Just for understanding: why do you throw an assert if name is in range [0, Count)?